cmd/kops/get_cluster.go

```diff
@@ -121,6 +136,8 @@ func RunGetClusters(context Factory, out io.Writer, options *GetClusterOptions)
 		if err != nil {
 			return err
 		}
+
+		fmt.Fprint(out, get_cluster_full_warning)
```

Comment:
We can but that seems a little funky, we need a warning, regardless. We cannot just add an annotation without a warning and instructions on how to remove